pages/index.vue

```diff
@@ -20,9 +20,19 @@
 </template>
 
 <script setup lang="ts">
-import { useNuxtApp } from '#app'
+import { useScreenOrientation } from '@vueuse/core'
```

Comment:
nice! this was something I was looking to fix and you beat me to it 😆 here's some thoughts

## design improvements
- removing library -  I'm fine to use this library, but if there's a snippet option that doesn't require us to add a dependency (each one adds risk/maintenance)
- resuability - This orientation check will likely be used throughout the app, so let's make this a [composable](https://vuejs.org/guide/reusability/composables.html#mouse-tracker-example) that can be reused easily anywhere in the app. This should also prevent other devs from needlessly creating additional `watch`s for orientation.

Reply:
I thought about a way I can try to refactor without the library. I added it mostly due to it changing exactly the way we wanted. But I have another idea. And I'll work on making a composable...Monday